_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -60448,6 +60448,10 @@
 /obj/machinery/airalarm/directional/south,
 /turf/open/floor/iron,
 /area/service/janitor)
+"thJ" = (
+/obj/machinery/holopad,
```

Comment:
Use `/obj/machinery/holopad/secure` (or whatever the path is). Heads of staff can force non-secure holopads to pick up when they call them, and security is intended to have secure holopads.

Reply:
Good catch, wasn't sure of the difference. I'll fix it in a bit.